### DISCUSSION TITLE
Welcome to HACTOBERFEST_2023 Discussions!

### DISCUSSION THREAD

**Discussion by ratnesh-maurya** [ratnesh-maurya/your_contribution#364] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ragav-47** [comment]
Hii i'm ragav and this is my first time contributing for open source....i would like to contribute here in this repo but i would like to know whether i can add a new programming language(C#)

**Comment by priyankaa-2004** [comment]
Hello I am Priyanka and I am new to hacktober fest. Can I contribute in this repository in Hacktober 2023?

**Reply by ratnesh-maurya** [reply]
Hello Priyanka You can contribute in this repo for Hacktoberfest 2023
